client/src/leap/soledad/client/sqlcipher.py

```diff
@@ -676,7 +676,7 @@ def close(self):
         Close the syncer and syncdb orderly
         """
         # close all open syncers
-        for url in self._syncers:
+        for url in self._syncers.keys():
```

Comment:
Try that code:
d = {1:1, 2:2, 3:3}
for i in d: del d[i]
(raises RuntimeError)
Now try that:
d = {1:1, 2:2, 3:3}
for i in d.keys(): del d[i]
Works! :dart: